index.html

```diff
@@ -22,9 +21,6 @@
   <audio src="deathsoun.mp3" id="death-audio" controls loop>
   <audio src="winsong.mp3" id="win-audio" controls loop>
   <audio src="hop.wav" id="hop-audio" controls>
-
   <script src="main.bundle.js"></script>
-  <!-- <script src="lib/index.js"></script> -->
-  <script src="lib/underscore-min.js"></script>"
```

Comment:
I was trying to use the underscore library to solve the frog hopping problem. it seems ridiculous now.